test/alice/handlers/help_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule Alice.Handlers.HelpTest do
   alias Alice.Handlers.TestHandler
 
   test "help_for_handler generates the correct output" do
-    assert Help.help_for_handler(TestHandler) ==
+    assert Help.help_for_handler(TestHandler, %{slack: %{me: %{name: "alice"}}}) ==
```

Comment:
@adamzaninovich not a fan of this hard-coded conn. wasn't sure if there was a cleaner way to do this in tests?

Reply:
maybe just use `Conn.new` and pass in the slack component

Reply:
Yea I like, that at least hides the use of the `slack` key to the internals until that gets changed with the whole different backend stuff.